invisible_cities/reco/paolina_functions_test.py

```diff
@@ -701,6 +702,11 @@ def test_paolina_functions_with_hit_energy_different_from_default_value(requeste
 
     assert not np.allclose(energies, energies_c)
 
+    energies_c = [v.E for v in voxels_c]
+    e_thr = min(energies_c) + fraction_zero_one * (max(energies_c) - min(energies_c))
+    # Test that this function doesn't fail
+    mod_voxels_c = drop_end_point_voxels(voxels_c, e_thr, min_vxls=0)
+
```

Comment:
maybe also show that the energy of voxels are equal to the sum of corresponding hit energies depending on energy_type?